backend/src/main/java/com/hyooun/wilson/review/domain/Change.java

```diff
@@ -1,17 +1,21 @@
 package com.hyooun.wilson.review.domain;
 
+import jakarta.persistence.Embeddable;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 
 @Getter
+@Embeddable
+@NoArgsConstructor
```

Comment:
Change 클래스에 @Embeddable 및 @NoArgsConstructor 애너테이션을 추가합니다.

```suggestion
@Embeddable
@NoArgsConstructor
```